src/AspNet.Security.OAuth.Autodesk/AutodeskAuthenticationDefaults.cs

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Builder;
+
+namespace AspNet.Security.OAuth.Autodesk
+{
+    /// <summary>
+    /// Default values used by the Autodesk authentication middleware.
+    /// </summary>
+    public static class AutodeskAuthenticationDefaults
+    {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationOptions.AuthenticationScheme"/>.
+        /// </summary>
+        public const string AuthenticationScheme = "Autodesk";
+
+        /// <summary>
+        /// Default value for <see cref="RemoteAuthenticationOptions.DisplayName"/>.
+        /// </summary>
+        public const string DisplayName = "Autodesk";
+
+        /// <summary>
+        /// Default value for <see cref="AuthenticationOptions.ClaimsIssuer"/>.
+        /// </summary>
+        public const string Issuer = "Autodesk";
+
+        /// <summary>
+        /// Default value for <see cref="RemoteAuthenticationOptions.CallbackPath"/>.
+        /// </summary>
+        public const string CallbackPath = "/signin-autodesk";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.AuthorizationEndpoint"/>.
+        /// </summary>
+        public const string AuthorizationEndpoint = "https://developer.api.autodesk.com/authentication/v1/authorize";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.TokenEndpoint"/>.
+        /// </summary>
+        public const string TokenEndpoint = "https://developer.api.autodesk.com/authentication/v1/gettoken";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.UserInformationEndpoint"/>.
+        /// </summary>
+        public const string UserInformationEndpoint = "https://developer.api.autodesk.com/userprofile/v1/users/@me";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.Scope"/>.
+        /// </summary>
+        public static readonly IEnumerable<string> Scope = new[] { "user-profile:read" };
```

Comment:
Please consider moving `user-profile:read` to a dedicated `Scopes` nested class in `AutodeskAuthenticationConstants`.